zerver/lib/create_user.py

```diff
@@ -90,14 +96,12 @@ def create_user(email: str, password: Optional[str], realm: Realm,
     user_profile = create_user_profile(realm, email, password, active, bot_type,
                                        full_name, short_name, bot_owner,
                                        is_mirror_dummy, tos_version, timezone)
-    if is_realm_admin:
-        user_profile.role = UserProfile.ROLE_REALM_ADMINISTRATOR
-    if is_guest:
-        user_profile.role = UserProfile.ROLE_GUEST
     user_profile.avatar_source = avatar_source
     user_profile.timezone = timezone
     user_profile.default_sending_stream = default_sending_stream
     user_profile.default_events_register_stream = default_events_register_stream
+    if role is not None:
+        user_profile.role = role
```

Comment:
I think this might read better as just having `role: int = UserProfile.ROLE_MEMBER` in the parameters?

Reply:
Let's leave this as a follow-up cleanup; I think it might end up wanting some replumbing; e.g. maybe this should be a thing we pass into `create_user_profile`?  Not sure.